src/noit_filters.c

```diff
@@ -623,6 +625,9 @@ noit_console_rule_configure(mtev_console_closure_t ncct,
   }
   rv = 0;
  bail:
+  if (add_arg_initialized) {
```

Comment:
Can lose the wrapping `if`

Reply:
Already merged this, but I'll go ahead and make these changes/push to master.

Reply:
`add_arg` might have been set to `fsnode`, if the release(of both `add_arg` and `fsnode`) is unconditional it seems like this function will unlock an extra time

Reply:
Yeah, that's why I did it this way.

I think it's probably safest to just leave it like this for now... if we rearrange, I'll do a separate PR.